### PR TITLE
VACMS-8049: consolidate facility related views displays

### LINKED ISSUE #8049: Move Facility related views displays from content view to Facilities view in #7500
## User Story or Problem Statement

The number of displays attached to the content view have grown substantially, to the point where the view is starting to feel a bit crowded. In #7500 a new Facilities view with three displays was created. We should move any displays related to facilities from the content view to this new Facilities view.

**Displays to be moved**

- [x] Facilities: https://prod.cms.va.gov/admin/content/facilities
- [x] Facilities Export (corresponds to the above facilities view display)
- [x] Facility Status: https://prod.cms.va.gov/admin/content/facility-status
- [x] VAMC System Service Audit: https://prod.cms.va.gov/admin/content/audit/vamc-system-service-audit
- [x] VAMC System Service Audit Export (Corresponds to the above view display)
- [x] VAMC EHR: https://prod.cms.va.gov/admin/content/audit/vamc-ehr

**Redirects**

Any displays receiving a new URL will need a redirect created for it so users visiting the old URLs end up at the new one

- [x] admin/content/facility-status => admin/content/facilities/facility-status
- [x] admin/content/audit/vamc-system-service-audit => admin/content/facilities/vamc-system-service-audit
- [x] admin/content/audit/vamc-ehr => admin/content/facilities/vamc-systems

### CMS Team

Please check the team(s) that will do this work. 

- [ ] `CMS Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide CMS Team ` (leave Sitewide unchecked and check the specific team instead)
  - [ ] `⭐️ Content ops`
  - [ ] `⭐️ CMS experience`
  - [ ] `⭐️ Offices`
  - [x] `⭐️ Product support`
  - [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by JayDarnell**
## Description

Closes #8049 

## Testing done

Visual Testing

## Screenshots

**Facilities (formerly a display within the content view)**
Screen Shot 2022-02-22 at 3 59 08 PM

**Facility Status (formerly a display within its own view called Facility Governance)**
Screen Shot 2022-02-22 at 3 59 33 PM

**VAMC systems (formerly a display called VAMC EHR within the content view - repurposed by Kev)**
Screen Shot 2022-02-22 at 3 59 44 PM

**VAMC System Service Audit (formerly a display within the content view)**
Screen Shot 2022-02-22 at 3 59 55 PM

## QA steps

The bulk of this work revolves around recreating view displays within the facility_services view that were previously located elsewhere. Most views were copied over as is with one exception. The VAMC EHR view was updated to match a specification provided by @kevwalsh within a testing environment he had set up. This view was renamed and updated to serve a slightly different purpose.

As an admin user:
- [x] Verify that admin/content/facility-status **redirects to** admin/content/facilities/facility-status
- [x] Compare admin/content/facilities/facility-status to https://prod.cms.va.gov/admin/content/facility-status (test some filters side by side and be sure the results match - be mindful of new content in production that may not be in tugboat)
- [x] Verify that admin/content/audit/vamc-system-service-audit **redirects to** admin/content/facilities/vamc-system-service-audit
- [x] Compare admin/content/facilities/vamc-system-service-audit to https://prod.cms.va.gov/admin/content/audit/vamc-system-service-audit (test some filters side by side and be sure the results match - be mindful of new content in production that may not be in tugboat)
- [x] Verify that admin/content/audit/vamc-ehr **redirects to** admin/content/facilities/vamc-systems
- [x] Compare admin/content/facilities/vamc-systems to https://main-g0ovfooigfojcgcoaykq1ghkb0vwg4wd.demo.cms.va.gov/admin/content/audit/vamc-systems (test soem filters side by side and be sure the results match - be mindful of new content in production that may not be in tugboat)
- [x] Access permissions for each view should be unchanged

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide CMS Team`
  - [ ] `⭐️ Content ops`
  - [ ] `⭐️ CMS Experience`
  - [ ] `⭐️ Offices`
  - [x] `⭐️ Product Support`

### CODE REVIEW

**Comment by swirtSJW** [comment]
I just resolved a composer lock conflict and did a force push.  Hopefully it should build now.

**Comment by swirtSJW** [comment]
Resolved conflict....awaiting tests...again,

**Comment by JayDarnell** [comment]
Looks like an accessibility failure @swirtSJW

**Comment by swirtSJW** [comment]
Rebased.   (fingers crossed)

File: docroot/modules/custom/va_gov_db/va_gov_db.install [hunk 1]
```diff
@@ -742,3 +772,20 @@ function va_gov_db_update_8033(&$sandbox) {
 
   return $messages;
 }
+
+/**
+ * Add redirects for restructured facility views.
+ */
+function va_gov_db_update_8034(&$sandbox) {
+  $messages = '';
+  $redirects = [
```
Comment (swirtSJW) [inline docroot/modules/custom/va_gov_db/va_gov_db.install:781]: Nice work on this Jay.

**Review by swirtSJW — APPROVED** [review]
Outstanding Jay.    Well thought out, well tested, and great QA steps.

**Pull request merged**